packages.gradle

```diff
@@ -115,7 +115,7 @@ def getPkgNameSuffix(component, type) {
 def getParentDir(bigtopBaseVersion, type) {
```

Comment:
#1286 seems to depend on this. How about making separate JIRA/PR to update the packages.gradle? Since deploying Hadoop depends ZooKeeper, #1286 should be addressed before this. @JiaLiangC.

Reply:
Ok, thanks for help review, i will make a  separate PR to update the packages.gradle